localization/english/gui/ancient_magic_title_view_l_english.yml

```diff
@@ -1,3 +0,0 @@
-﻿l_english:
```

Comment:
Because the button for granting to a mage was removed, as was the effect. Grant To Mage was only there as a workaround to make sure a magocratic character was appointed to lead a city of magic, but it never actually worked.